### ISSUE TITLE
Swap RA axis on plots

### ISSUE THREAD

**Issue by hombit** [MickaelRigault/ztffields#2]
Currently RA direction is not how we usually use it in astronomy, since we look to the stellar flare from the inside: RA should grow from right to left. See ligo's skymap package for the reference
https://lscsoft.docs.ligo.org/ligo.skymap/plot/allsky.html

I'd happy to replace our code with yours, see what we did in this paper, fig. 1
https://ui.adsabs.harvard.edu/abs/2023A%26A...672A.111P/abstract

**Comment by MickaelRigault** [comment]
Hi @hombit I'll have a look thanks for that, thanks !

**Comment by MickaelRigault** [comment]
I might be wrong (it is late) but:
- here: https://www.wis-tns.org/
or 
- https://irsa.ipac.caltech.edu/data/ZTF/docs/releases/dr07/ztf_release_notes_dr07.pdf
or here, for LSST sims:
- https://www.lsst.org/sites/default/files/mw.png
RA increases rightward.

However with RA=0 in the center, not sure this si what I do.

Anyway, I'll look into it

**Comment by hombit** [comment]
Hi @MickaelRigault. You are right, there are a lot of examples how people did it wrong (BTW ZTF DR page did it right, check again), we can discuss the reasons here, but the most important technical reason is that almost all spherical projection software was developed with geographical, not astronomical, needs in mind. `matplotlib`, for instance,  doesn't provide any simple way to flip latitude axis (however `astropy.visualisation` [does](https://docs.astropy.org/en/stable/visualization/wcsaxes/index.html)).

It is not the matter of where RA=0 is located, it is the matter of where your north and south poles are. If north is up, RA grows left.

All astronomical software, sky maps, finding charts, etc, do look on sky from inside the stellar sphere, not outside. The only exception I know is [sky globes](https://commons.wikimedia.org/wiki/File:Celestial_Sphere_-_Equatorial_Coordinate_System.png), for obvious reasons.
- Digital sky atlases: [Aladin](https://aladin.cds.unistra.fr/AladinLite/), [Stellarium](http://stellarium.org), [Legacy survey viewer](https://www.legacysurvey.org/viewer), [Vizier Mine](https://vizier.cds.unistra.fr/vizier/VizieR/vizmine/vizMine.gml)
- Paper atlases: [Celestial Atlas by Jamieson](https://commons.wikimedia.org/wiki/Category:A_Celestial_Atlas_by_Alexander_Jamieson)
- FITS image viewers: DS9/[JS9](https://js9.si.edu)
- Finding charts: [Swift](https://swift.gsfc.nasa.gov/uvot_tdrss/uvot_std_products.html)

**Comment by MickaelRigault** [comment]
This is now fixed with version 0.6


you can specify the `system` you want. by default it will now be **astro** while it used to be **geo**.

Shifting the origin is now fixed. It used to be broken.
```python
import ztffields
fieldid = ztffields.get_fieldid(galb_range=[[10,90],[-20,-10]])
_ = ztffields.skyplot_fields(fieldid, origin=0, system="astro radec")
```

![image](https://github.com/MickaelRigault/ztffields/assets/7812822/c8b406de-6eb1-4db1-b38a-ede646243228)


use the explicit `origin=180, system="geo radec"` options to recover the original behavior.
```python
_ = ztffields.skyplot_fields(fieldid, origin=180, system="geo radec")
```
![image](https://github.com/MickaelRigault/ztffields/assets/7812822/e5bcad0e-80a3-41e7-81fb-111e422e45d3)

**Comment by MickaelRigault** [comment]
@hombit if you can update ztffields and make sure that is indeed behaving correctly, I'll close the issue

**Comment by hombit** [comment]
@MickaelRigault thank you so much!

**Comment by MickaelRigault** [comment]
closing it now. Implemented in v=0.6.0.

Re-open if needed
